src/screens/EventDetailsScreen/index.tsx

```diff
@@ -70,29 +72,34 @@ const EventDetailsScreen: React.FC<TEventDetalsScreenProps> = ({ route }) => {
 
   if (isBMPlayerShowing) {
     return (
-      <Player
-        autoPlay
-        configuration={{
-          url: 'https://video-ingestor-output-bucket.s3.eu-west-1.amazonaws.com/6565/manifest.m3u8',
-          poster:
-            'https://actualites.music-opera.com/wp-content/uploads/2019/09/14OPENING-superJumbo.jpg',
-        }}
-        title="event title"
-        subtitle="some event subtitle"
-        onClose={closePlayer}
-        analytics={{
-          videoId: 'blahblahblah',
-          title: 'Some video title',
-          experiment: 'ROH TV app',
-          customData1: '',
-          customData2: '',
-          customData3: '',
-          customData4: '',
-          customData5: '',
-          customData6: '',
-          customData7: '',
-        }}
-      />
+    <View style={styles.root}>
```

Comment:
Can we wrap this in a platform check, rather than commenting it out? Keen that we get to a cross-platform codebase ASAP, even if chunks don't work for both platforms.

Reply:
Hmm, yes, that was my initial plan but doing so causes the runtime error that was the original issue. May I suggest that we prioritise implementation of the bitmovin player so as to get rid of this problem? I may be able to lift the platform check up into the player component though, that might work actually.

Reply:
Found and checked in a solution for platform specific player files, which sets us up nicely to proceed with the player implementation...